client/homebrew/editor/metadataEditor/metadataEditor.less

```diff
@@ -42,6 +42,9 @@
 		&>.value{
 			flex           : 1 1 auto;
 			width          : 50px;
+			&:invalid {
+				background : #ffb9b9;
+			}
```

Comment:
Is the `invalid` state only communicated by color?
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

Reply:
It also pops the error message as soon as the error happens, which fades away after about 5 long seconds. Then, if you continue typing it will show that error message again. See the screenshot in parent comment. 

So it’s pretty “in your face” about it. But it’s all handled by the browser and only this little extra but is needed in the css.